packages/mux-player/src/index.ts

```diff
@@ -309,7 +312,7 @@ class MuxPlayerElement extends VideoApiElement {
     newValue: string
   ) {
     super.attributeChangedCallback(attrName, oldValue, newValue);
-    this.#render();
+    this.#render({ [camelCase(attrName)]: newValue });
```

Comment:
blocking: let's break this out into a function for translating attribute names -> prop names